packages/flutter_webrtc/README.md

```diff
@@ -0,0 +1,47 @@
+# flutter_webrtc_tizen
+
+[![pub package](https://img.shields.io/pub/v/flutter_webrtc_tizen.svg)](https://pub.dev/packages/flutter_webrtc_tizen)
+
+The Tizen implementation of [`flutter_webrtc`](https://github.com/flutter-webrtc/flutter-webrtc).
+
+## Required privileges
+
+To use this plugin in a Tizen application, you may need to declare the following privileges in your `tizen-manifest.xml` file.
+
+```xml
+<privileges>
+    <privilege>http://developer.samsung.com/privilege/camera</privilege>
+    <privilege>http://tizen.org/privilege/internet</privilege>
+    <privilege>http://tizen.org/privilege/camera</privilege>
+    <privilege>http://tizen.org/privilege/recorder</privilege>
+</privileges>
+```
+
+- The internet privilege (`http://tizen.org/privilege/internet`) is required to access the internet.
+- The camera priviege (`http://tizen.org/privilege/internet`) is required to use camera.
+- The recorder priviege (`http://tizen.org/privilege/recorder`) is required to record video and audio.
```

Comment:
```suggestion
- The camera privilege (`http://tizen.org/privilege/camera`) is required to use camera.
- The recorder privilege (`http://tizen.org/privilege/recorder`) is required to record video and audio.
```

Can a user declare only `http://developer.samsung.com/privilege/camera` or should they also declare `http://tizen.org/privilege/internet` to achieve full functionality on TV devices?

Reply:
Solved

Reply:
Do you have an answer to the question?

Reply:
If we want to use camera on TV devices, we just need add  http://developer.samsung.com/privilege/camera.

If we want to use camera on RB5 devices, we just need add  add http://tizen.org/privilege/camera.

Reply:
It'd be better if the README becomes clear about the fact, to avoid any confusion. It doesn't matter much though.

Reply:
I have updated README file, separately lists the privileges required for TV device and other device, I think it more clear.